jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java

```diff
@@ -165,19 +176,27 @@ public List<Snippet> getSnippets() {
      */
     public List<CompletionItem> getCompletionItem(final Range replaceRange, final String lineDelimiter,
             boolean canSupportMarkdown, List<String> context) {
+        List<Snippet> snippets = getSnippets();
+        Map<String, String> values = new HashMap<String, String>();
+        int size = context.size();
+        if (size == snippets.size() + 2) { // the last 2 strings are package name and class name
+            values.put(PACKAGE_NAME, context.get(size - 2));
+            values.put(CLASS_NAME, context.get(size - 1));
```

Comment:
Should we remove these from the snippet list before streaming?

Reply:
Before streaming snippets? are you talking about removing package and class name from context list?

Reply:
Yes from the context list

Reply:
why? they won't impact to the following usage of the list, not necessary to do extra removal.

Reply:
A suggestion to make the implementation cleaner, but if you feel it is not needed that is okay